### PR TITLE
Added a page called how to create & submit a PR under About section

### PR DESCRIPTION
**Pull request by bellex3**
Created a page called How to Create & Submit a Pull Request (PR) under the About section.

### CODE REVIEW

File: docs/about/how-to-create-and-submit-PR.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+**SECOND APPROACH**
+
+Another way of merging your changes is using Mergify.
+
+**Pros of using Mergify**
+-   Don't have to manually merge main and can ensure that your code passes the tests before merging. Especially if you forget to pull the latest from main, you may come across tests breaking. Mergify helps you here.
+
+**Con of using Mergify**
+-   Can take a while, especially if there are multiple merges happening at the same time. Mergify has a queue of jobs that it tries to merge in order so you may be blocked by other merges. Nonetheless, it is very seamless and simple.
```
Comment (YanZ777) [inline docs/about/how-to-create-and-submit-PR.md:96]: **suggestion:** I've also found Mergify to get a bit too merge happy when getting an approval, i.e. it merges things that only have design review when the PR should really have both design and code review. Could we mention this as well?

**Review by YanZ777 — APPROVED** [review]
I have some minor suggestions, none of them blockers though.
Thanks for making this documentation accessible to everyone! It looks really nice with all the screenshots and emojis.

File: docs/about/how-to-create-and-submit-PR.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+**Step 8:**  First things first, gotta give your PR a title. There is a required naming convention for the title: `[Jira ticket number]-brief description of task`. Brackets needed.
+
+Ex: `[MB-13507] Changed AOA language for Customer PPM Form`
+
+a) Next, provide the link to your Jira ticket.
+b) Under the “Summary” section, this is where you may add in any special details you believe reviewers should be aware of. For example, explaining why you wrote some code the way you did, why you chose this approach instead of another.
+c) “Setup to Run Your Code”-thankfully there is already boilerplate info there, so no need to touch this section.    
+d) Under “Additional Steps”, this is where you add the steps reviewers need to take to view and check your changes.
+e) The “Verification Steps for Author” section is where you check to ensure your PR is ready for review. (See the **Final Touches** section below for more info on requesting reviewers). 
+f) “Verification Steps for Reviewers”- section for reviewers, don’t touch.
```
[the remark this answers is not in the document]
Comment (bellex3) [reply docs/about/how-to-create-and-submit-PR.md]: @YanZ777 done. I have added this info in

File: docs/about/how-to-create-and-submit-PR.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+**Step 9**:  On the right hand side, you’ll see some categories that you need to assign values to. Specifically, we will focus on the “Reviewers” and “Labels”.
+
+![Image showing the categories side panel](/img/create_pr/categories_side_panel.png)
+
+a) Under “Reviewers”, select your team from the dropdown. If you are making a change to the Frontend that also affects the layout/design of the app (i.e. adding a new form on a page), you’ll need to also select the “truss-design” team. Similarly, if you are making a change to the database, you’ll also need to select the “truss-db” team.
+
+![Image showing the reviewers side panel](/img/create_pr/reviewers.png)
+
+At least 1 engineer from your team needs to approve your PR in order for you to merge it in.<br/>
+If you also need design/database review, one person from those teams will need to approve your PR as well.
```
[the remark this answers is not in the document]
Comment (bellex3) [reply docs/about/how-to-create-and-submit-PR.md]: @YanZ777 done. I have added this info in

File: docs/about/how-to-create-and-submit-PR.md (hunk 1 above)
Comment (bellex3) [reply docs/about/how-to-create-and-submit-PR.md:97]: @YanZ777 done. I have added this info in. I also made a note that the person should add the "ready-to-merge" label only after they know they've gotten all the necessary approvals and reviews

**Pull request merged**